deployments/goerli/weth/deploy.ts

```diff
@@ -0,0 +1,160 @@
+        signer.address,                    // Blacklister
+        signer.address                     // Owner
+      ));
+    }
+  );
+
+  // Deploy stETH / ETH SimplePriceFeed
+  const stETHtoETHPriceFeed = await deploymentManager.deploy(
+    'stETHToETH:simplePriceFeed',
+    'test/SimplePriceFeed.sol',
+    [
+      exp(0.98882408, 18), // Latest answer on mainnet at block 16170924
+      18
+    ]
+  );
+
+  // Deploy cbETH / ETH SimplePriceFeed
+  const cbETHtoETHPriceFeed = await deploymentManager.deploy(
+    'cbETHToETH:simplePriceFeed',
+    'test/SimplePriceFeed.sol',
+    [
+      exp(0.97, 18),
+      18
+    ]
+  );
+
+  // Deploy WstETHPriceFeed
+  const wstETHPriceFeed = await deploymentManager.deploy(
+    'wstETH:priceFeed',
+    'WstETHPriceFeed.sol',
+    [
+      stETHtoETHPriceFeed.address,                  // stETH / ETH price feed
+      wstETH.address,                               // wstETH
+      8                                             // decimals
+    ]
+  );
+
+  // Deploy constant price feed for WETH
+  const wethConstantPriceFeed = await deploymentManager.deploy(
+    'WETH:priceFeed',
+    'ConstantPriceFeed.sol',
+    [
+      8,                                             // decimals
+      exp(1, 8)                                      // constantPrice
+    ]
+  );
+
+  // Deploy scaling price feed for cbETH
+  const cbETHScalingPriceFeed = await deploymentManager.deploy(
+    'cbETH:priceFeed',
+    'ScalingPriceFeed.sol',
+    [
+      cbETHtoETHPriceFeed.address,                  // cbETH / ETH price feed
+      8                                             // decimals
+    ]
+  );
+
+  // Deploy all Comet-related contracts
+  const deployed = await deployComet(deploymentManager, deploySpec);
+  const { comet } = deployed;
+
+  // Deploy Bulker
+  const bulker = await deploymentManager.deploy(
+    'bulker',
+    'bulkers/MainnetBulker.sol',
+    [
+      comet.governor(),        // admin_
+      WETH.address,            // weth_
+      wstETH.address           // wsteth_
+    ]
+  );
+
+  return { ...deployed, bulker, fauceteer, fxRoot };
+}
+
+async function mintTokens(deploymentManager: DeploymentManager) {
+  const signer = await deploymentManager.getSigner();
+  const contracts = await deploymentManager.contracts();
+  const fauceteer = contracts.get('fauceteer')!;
+
+  debug(`Attempting to mint as ${signer.address}...`);
+
+  // If we haven't spidered new contracts (which we could before minting, but its slow),
+  //  then the proxy contract won't have the impl functions yet, so just do it explicitly
+  const cbETHProxy = contracts.get('cbETH')!, cbETHImpl = contracts.get('cbETH:implementation')!;
+  const cbETH = cbETHImpl.attach(cbETHProxy.address);
+  await deploymentManager.idempotent(
+    async () => (await cbETH.balanceOf(fauceteer.address)).eq(0),
+    async () => {
+      debug(`Minting 10K cbETH to fauceteer`);
+      const amount = exp(10_000, await cbETH.decimals());
```

Comment:
Is this amount good?

Reply:
If it doesn't require goerli ETH then maybe a larger number?

Reply:
If it doesn't require goerli ETH then maybe a larger amount? It should be fine

Reply:
Ok increased the amount minted to 1mn cbETH. Each fauceteer `drip` would give roughly 100 ETH.